charts/argo-cd/templates/crds/crd-application.yaml

```diff
@@ -125,6 +125,11 @@ spec:
               sync:
                 description: Sync contains parameters for the operation
                 properties:
+                  autoHealAttemptsCount:
+                    description: SelfHealAttemptsCount contains the number of auto-heal
+                      attempts
+                    format: int64
+                    type: integer
```

Comment:
v2.13.1 has them. I wonder when we missed them. 🤔  
https://github.com/argoproj/argo-cd/blob/v2.13.1/manifests/crds/application-crd.yaml